R/covr.R

```diff
@@ -426,3 +456,31 @@ add_hooks <- function(pkg_name, lib, fix_mcexit = FALSE) {
 
   writeLines(text = lines, con = load_script)
 }
+
+## BEGIN Oracle Contribution
+# License: GPL-3 with additional permission for MIT under GPL-3 Section 7 as set forth in license documents.
+#
+# check if gcc or icc is used
+get_compiler <- function()
+{
+  mkconf <- readLines(file.path(R.home(component="etc"), "Makeconf"))
+  compiler <- rex::re_matches(mkconf,
+                rex::rex(start, "CC = ",
+                         capture(name = "compiler", anything),
```

Comment:
Should probably just be `rex::rex(start, "CC", any_spaces, "=", any_spaces, capture(name = "compiler", non_spaces))`